lib/declare_schema/schema_change/all.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module DeclareSchema
+  module SchemaChange
+    module All
+    end
+    autoload :Base, 'declare_schema/schema_change/base'
+    autoload :ColumnAdd, 'declare_schema/schema_change/column_add'
+    autoload :ColumnChange, 'declare_schema/schema_change/column_change'
+    autoload :ColumnRename, 'declare_schema/schema_change/column_rename'
+    autoload :ForeignKeyAdd, 'declare_schema/schema_change/foreign_key_add'
+    autoload :ForeignKeyRemove, 'declare_schema/schema_change/foreign_key_remove'
+    autoload :IndexAdd, 'declare_schema/schema_change/index_add'
+    autoload :IndexRemove, 'declare_schema/schema_change/index_remove'
+    autoload :PrimaryKeyChange, 'declare_schema/schema_change/primary_key_change'
+    autoload :TableAdd, 'declare_schema/schema_change/table_add'
+    autoload :TableChange, 'declare_schema/schema_change/table_change'
+    autoload :TableRemove, 'declare_schema/schema_change/table_remove'
+    autoload :TableRename, 'declare_schema/schema_change/table_rename'
```

Comment:
Loving how clean this looks with the `autoload`. I want to start using this pattern more where we can.

Reply:
Pretty slick, huh? I occasionally see Rails doing it this way.